kaldb/src/main/java/com/slack/kaldb/logstore/search/KaldbDistributedQueryService.java

```diff
@@ -126,25 +128,26 @@ private void updateStubs() {
     ScopedSpan span =
         Tracing.currentTracer().startScopedSpan("KaldbDistributedQueryService.distributedSearch");
 
-    try {
-      List<ListenableFuture<SearchResult<LogMessage>>> queryServers = new ArrayList<>(stubs.size());
-      span.tag("queryServerCount", String.valueOf(stubs.size()));
-
-      for (KaldbServiceGrpc.KaldbServiceFutureStub stub : stubs.values()) {
-        ListenableFuture<KaldbSearch.SearchResult> searchRequest =
-            stub.withDeadlineAfter(READ_TIMEOUT_MS, TimeUnit.MILLISECONDS)
-                .withInterceptors(
-                    GrpcTracing.newBuilder(Tracing.current()).build().newClientInterceptor())
-                .search(request);
-        Function<KaldbSearch.SearchResult, SearchResult<LogMessage>> searchRequestTransform =
-            SearchResultUtils::fromSearchResultProtoOrEmpty;
-        queryServers.add(
-            Futures.transform(
-                searchRequest, searchRequestTransform::apply, MoreExecutors.directExecutor()));
-      }
+    List<ListenableFuture<SearchResult<LogMessage>>> queryServers = new ArrayList<>(stubs.size());
+    span.tag("queryServerCount", String.valueOf(stubs.size()));
+
+    for (KaldbServiceGrpc.KaldbServiceFutureStub stub : stubs.values()) {
+      ListenableFuture<KaldbSearch.SearchResult> searchRequest =
+          stub.withDeadlineAfter(READ_TIMEOUT_MS - 100, TimeUnit.MILLISECONDS)
```

Comment:
Please extract LOCAL_QUERY_TIMEOUT_DURATION.toMillis() - 100 into a local variable and add a comment on why it's needed. Also, please document the magic number 100 that way we remember why we set it to that number later and what it can be updated to.

Reply:
Ah, these were inadvertently included from a previously staged commit and are not needed. I've removed both.